draft-ietf-quic-transport.md

```diff
@@ -833,7 +834,7 @@ consuming a large amount of memory.  To enable a receiver to limit memory
 commitments for a connection, streams are flow controlled both individually and
 across a connection as a whole.  A QUIC receiver controls the maximum amount of
 data the sender can send on a stream as well as across all streams at any time,
-as described in {{data-flow-control}} and {{fc-credit}}.
+as described in Sections {{<data-flow-control}} and {{<fc-credit}}.
```

Comment:
what does "<" do?

Reply:
New magic.  It is rendered as "4.3" instead of "Section 4.3".